routes/university/universitySecure.js

```diff
@@ -65,14 +65,14 @@ router.post('/', async (req, res) => {
 router.put('/', async (req, res) => {
     try {
         if (req.user.roleID == 14139 || req.user.roleID == 21149) {
-            if (req.body && req.body.id) {
+            if (req.body && req.body.id && req.body.updatedData) {

```

Comment:
Take a look at what the frontend is sending back -- https://github.com/ISTE-500-501-TeamWatchHill/frontend/blob/main/src/components/editpopup/editpopup.js#L228-L238

Doesn't match, so this would just break things. I do agree that we should've maybe stuffed the updated data in a child object, but I'm not sure this is worth changing now if it'll break the frontend.